pkgs/development/python-modules/lox/default.nix

```
@@ -0,0 +1,50 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+
+  pathos,
+  pytestCheckHook,
+  pytest-benchmark,
+  pytest-mock,
+  pytest-runner,
+  setuptools,
+  sphinx-rtd-theme,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "lox";
+  version = "0.12.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "BrianPugh";
+    repo = "lox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Iv3ZdfsvFLU6lhlH1n+eQ+KIrXESsnC1S2lVFnKFV08=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pathos
+    sphinx-rtd-theme # upstream package lists this as a runtime dep
+  ];
```

```suggestion
  pythonRemoveDeps = [ "sphinx_rtd_theme" ];

  dependencies = [ pathos ];
```
We can use `pythonRelaxDepsHook` to remove the wrong requirements.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook

---

Removed